spec/factories/users.rb

```diff
@@ -1,6 +1,8 @@
 FactoryGirl.define do
   factory :user do
+    name 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.